tests/helpers.py

```diff
@@ -67,6 +67,29 @@ def get_ddo(client, base_ddo_url, did):
         return None
 
 
+def get_chains_list(client, url):
+    rv = client.get(url + f"/list/", content_type="application/json")
+    try:
+        fetched_list = json.loads(rv.data.decode("utf-8"))
+        return fetched_list
+    except (Exception, ValueError) as e:
```

Comment:
please do not use Exception catching and print statements. Use assert statements. In test helpers, we should have clarity, not hide complexity by catching exceptions. This hides the cause of issues in tests and can not be debugged easily.

same for the other helper.

Reply:
actually I read further and I would get rid of these helpers entirely and make the tests call the list and status endpoints and using the response.json() as a basis for assertions. These 2 functions have no purpose from my point of view:
- they are not util functions, because they use the routes themselves
- they are not test helpers per se, since:
- they hide complexity
- they don't seem to be reusable, they are only used once, and don't help with any code duplication removal or design patterns.